### DISCUSSION TITLE
Variable size of the tab name - is this  feature of the bug?

### DISCUSSION THREAD

**Discussion by danieltomasz** [chrisgrieser/shimmering-focus#220] [Q&A] [answered]
Variable size of the tab name - is this  a feature of the bug? :) 

I there is only one tab in pane the title is much bigger

![image](https://user-images.githubusercontent.com/7980381/193916560-e37d52bb-89e0-4eb9-9fe5-1f7e26a5a1bd.png)

**Comment by chrisgrieser** [comment, marked as answer]
It's a feature, also prominently explained in the second screenshot of the readme ;)

You can turn it off in the style settings, it's called tab-bar-header-hybrid.
